api.raml

```diff
@@ -64,7 +64,7 @@ types:
         example: "developers"
       permission?:
         description: "The permission level for team members in the organization."
-        enum: [read, edit, admin]
```

Comment:
This is incorrect :) You can't change the permission to `owner` with a POST or PATCH request.

The `owner` value can only be returned on GET requests.